packages/addons/tsconfig.test.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig",
+  "compilerOptions": {
+    "sourceMap": true
+  },
+  "include": [ "__tests__/**/*.ts" ],
```

Comment:
**nitpick:** wrong folder. I don't understand how ts-jest is able to compile the test files while there are not part of a project!
```suggestion
  "include": [ "test/**/*.ts" ],
```